test/integration/add_artefact_test.rb

```diff
@@ -0,0 +1,21 @@
+require "integration_test_helper"
+
+class AddArtefactTest < ActionDispatch::IntegrationTest
+  setup do
+    setup_users
+    stub_linkables
+  end
+
+  should "create a new artefact" do
+    visit root_path
+    click_link "Add artefact"
+
+    fill_in "Name", with: "Thingy McThingface"
+    fill_in "Slug", with: "help/thingy-mc-thingface"
+    select "Help page", from: "Kind"
+
+    click_button "Save and go to item"
+
+    assert page.has_content?("Thingy McThingface")
```

Comment:
Nice. Could we check that the slug and kind have been set correctly too or is that overkill?